app/models/web_hook_events_daily_aggregate.rb

```diff
@@ -25,7 +25,7 @@ def aggregate!
         self.web_hook_id,
       )
 
-    self.mean_duration = events.sum(:duration) / events.count
+    self.mean_duration = events.sum(:duration) / events.count if events.count > 0
```

Comment:
is the default in the DB `0`? is it better to set the `mean_duration` to _something_ rather than skipping this entire line?

Reply:
https://github.com/discourse/discourse/blob/4262bdd4bdf3467f36c73cca0f60bd0e73df94bb/app/models/web_hook_events_daily_aggregate.rb#L44

I the mean duration is zero by default, so it is safe to assume that not running this code, the value there will be 0